### PR TITLE
Fix grouping for latest doxygen.

### PR DESCRIPTION
**Pull request by traud**
Since doxygen 1.8.16, opening and closing a group must not be done as C comment but as doxygen command. In other words, not one but two asterisk characters are required so that doxygen finds a group. This issue was found in another project initially: Opusfile. Then, I double-checked other Open Source projects. In case of popt, I am not sure this is the correct solution: 20 years ago, commit 02c43f2 introduced this issue. However, I am unsure what that group ‘popt’ is about.

Instead, what about removing all

- `\ingroup popt`
- `@{`
- `@}`

in other words, not using [doxygen grouping](https://www.doxygen.nl/manual/grouping.html) at all? If you prefer that, I change this Pull Request to that approach.

### CODE REVIEW

**Comment by traud** [comment]
Anything blocking this? It is just a syntactic change, in the comments to open up a Doxygen [special comment block](https://www.doxygen.nl/manual/docblocks.html). Should not harm anything. If you cannot decide on my question, let us simply merge this as is.

**Comment by pmatilai** [comment]
Sorry for the slow response. Back when the doxygen annotations were added, popt lived inside the rpm source tree (see https://github.com/rpm-software-management/rpm/commit/6aafc439daffd1232b410346abb2307cef0a2d4d) and the grouping probably made sense there.
As popt is a standalone project now, the grouping makes no sense whatsoever.

So I think just removing the grouping makes most sense...

**Comment by pmatilai** [comment]
...and done. Thanks for pointing this out!
